src/bin/patchmanager-daemon/patchmanagerobject.cpp

```diff
@@ -2969,3 +3007,113 @@ QString PatchManagerObject::pathToMangledPath(const QString &path, const QString
     qDebug() << Q_FUNC_INFO << "Path after mangle" << newpath;
     return newpath;
 }
+
+/*! Set up the filter parameters and fill it with some initial contents.
+ *
+ *  \sa PatchManagerFilter::setup()
+*/
+void PatchManagerObject::setupFilter()
```

Comment:
I would strongly recommend this be a separate cpp/h combo, since you already have the .h declaration separation